src/components/forms/editSiteForm/index.tsx

```diff
@@ -6,15 +6,20 @@ import { EditSiteRequest } from '../ducks/types';
 import { NEIGHBORHOOD_OPTS } from '../../../assets/content';
 import { Flex, FullInputNumber } from '../../themedComponents';
 import TitleStack from '../../titleStack';
+import styled from 'styled-components';
 
 interface EditSiteFormProps {
   readonly formInstance: FormInstance<EditSiteRequest>;
 }
 
+const EditForm = styled(Form)`
+  width: 100%;
+`;
+
 const EditSiteForm: React.FC<EditSiteFormProps> = ({ formInstance }) => {
   return (
     <>
-      <Form name="basic" form={formInstance}>
```

Comment:
❓ Does this still look okay in the other places (I think just site page) this form is used?